velocidi-sdk/src/main/kotlin/com/velocidi/FetchAdvertisingId.kt

```diff
@@ -0,0 +1,36 @@
+package com.velocidi
+
+import android.content.Context
+import java.util.*
+
+abstract class FetchAdvertisingId<T>(context: Context) {
```

Comment:
We shouldn't store Android context as class properties. It may lead to memory leaks.

To avoid that, we need to pass context to `FetchAdvertisingId` which will execute `GetAdvertisingIdTask` with this context.